### PR TITLE
change the way the name is generated.

### PR DESCRIPTION
**Pull request by jonhartzler**
added a semicolon between the titles when they are run together.  It may be that only the title line 1 is necessary but for now lets go both.

### CODE REVIEW

**Comment by danthewolfe** [comment]
:+1:

File: app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx [hunk 1]
```diff
@@ -37,13 +37,25 @@ var ShowcaseTitleBar = React.createClass({
       lineHeight: titleHeight + "px",
     };
   },
+  name: function () {
+    if (this.props.showcase.name_line_2) {
+      return this.props.showcase.name_line_1 + this.nameSpacer() + this.props.showcase.name_line_2;
+    }
+    return this.props.showcase.name_line_1;
+  },
+  nameSpacer: function () {
```
Comment (jaronkk) [inline app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx]: Can you add a newline between these two functions?

File: app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx [hunk 2]
```diff
@@ -37,13 +37,25 @@ var ShowcaseTitleBar = React.createClass({
...
+    return "";
+  },
 
   render: function() {
     if (this.props.showcase) {
       return (
         <div className="showcases-title-bar" id="showcases-title-bar" style={this.style()}>
-          <h2 className="showcases-title-bar-title overflow-ellipsis" style={this.titleStyle()} title={this.props.showcase.name}>
-          <span className="title">{this.props.showcase.name_line_1}</span>&nbsp;
+          <h2 className="showcases-title-bar-title overflow-ellipsis" style={this.titleStyle()} title={this.name}>
```
Comment (jaronkk) [inline app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx]: `this.name` should be `this.name()`, right?

File: app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx [hunk 3]
```diff
@@ -37,13 +37,25 @@ var ShowcaseTitleBar = React.createClass({
       lineHeight: titleHeight + "px",
     };
   },
+  name: function () {
```
Comment (jaronkk) [inline app/assets/javascripts/components/showcase/ShowcaseTitleBar.jsx]: A newline here as well

**Comment by jaronkk** [comment]
:+1:

**Pull request merged**